src/viur_cli/local.py

```diff
@@ -229,3 +233,86 @@ def show_output_if_not(args, check_str):
                     all_checks_passed = False
 
     return all_checks_passed
+
+
+def checknpm(autofix):
+    """
+    Check for npm vulnerabilities in the project and optionally fix them.
+
+    This function runs the "npm audit" command to check for vulnerabilities in the project's dependencies. If any
+    vulnerabilities are found, the function prompts the user to view the details and optionally run "npm audit fix
+    --force" to automatically fix the vulnerabilities.
+
+    Args:
+        autofix (bool): A boolean indicating whether to automatically fix vulnerabilities.
+
+    Raises:
+        subprocess.CalledProcessError: If an error occurs while running the "npm audit" or "npm audit fix" commands.
+    """
+    sources_folder = './sources'
+
+    try:
+        # Run "npm audit" command and capture the output
+        result = subprocess.run(
+            ['npm', 'audit', '--json'],
+            capture_output=True,
+            cwd=sources_folder,
+            encoding='utf-8'
+        )
+
+        vulnerabilities = json.loads(result.stdout)["metadata"]["vulnerabilities"]
+
+        if vulnerabilities["total"] >= 0:
+
+            if not autofix:
+                print(
+                    f'Npm found {vulnerabilities["total"]} Vulnerabilities \n'
+                    f'info: {vulnerabilities["info"]}\n'
+                    f'low: {vulnerabilities["low"]}\n'
+                    f'moderate: {vulnerabilities["moderate"]}\n'
+                    f'high: {vulnerabilities["high"]}\n'
+                    f'critical: {vulnerabilities["critical"]}\n'
+                    f'total: {vulnerabilities["total"]}\n'
+                )
+                show_vulnerabilities = input('Do you want a list of the found Vulnerabilities? (y/N)').strip().lower()
+
+                if show_vulnerabilities == 'y':
+                    pprint(json.loads(result.stdout)["vulnerabilities"])
```

Comment:
You can save the second `json.loads`-call
```suggestion
                    pprint(audit["vulnerabilities"])
```